Feedback/templates/Feedback/index_col.html

```diff
@@ -0,0 +1,721 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+      <meta charset="utf-8">
+      <title>College Work</title>
+      <link rel="stylesheet" href="http://netdna.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+      <style>
+      body{background: url(http://digitallearning.eletsonline.com/wp-content/uploads/2018/04/Mettl-partners-with-NSDC-to-innovate-assessment-technology.jpg);
+         background-size:cover;
+         background-repeat:no-repeat;
+         opacity: 0.7;
+       padding: 30px 30px 0px 30px;}
```

Comment:
Please correct indentation.

Reply:
Ok sure will do asap, Thanks for the suggestions.